assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -2284,6 +2287,14 @@ protected String build_student_confirm_submission_context(VelocityPortlet portle
             }
         }
 
+        User submitter = (User) state.getAttribute("student");
+        if (submitter == null) {
+            submitter = user;
+        }
+        context.put("submittee", submitter);
```

Comment:
What is a submittee? Somebody who receives a submission? I'm confused what this means. You should also put this in a constant, maybe with a comment explaining what it is.